src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,17 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private static final User[] users = new User[]{
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
     };
 
-    /**
-     * Find user by email. All users are stored in <code>private static final User[] users</code>
-     * @param email - the input parameter
-     * @return - user if his email is equal to passed email.
-     * Return <code>null</code> if there is no suitable user
-     */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
Please rework solution without static methods. Create object of required service as class variable.
Static methods have worse maintainability and harder to test unless they have utility nature
```suggestion
    public User findByEmail(String email) {
```